build.gradle

```diff
@@ -332,39 +332,36 @@ allprojects { currentProj ->
       formatAnnotations()
     }
 
-    // Only define a groovyGradle task on the root project, for simplicity in setting the target pattern
-    if (currentProj == currentProj.rootProject) {
```

Comment:
Not sure removing this check is right, in that I'm not sure all `build.gradle` files in nested directories will still be formatted.  Is the issue here that the top-level `build.gradle` file was not getting formatted?

Also if I recall correctly tweaking these settings can have significant impacts on incremental build performance so we should test that.